lib/Pages/UserPage.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:convex_bottom_bar/convex_bottom_bar.dart';
+import 'package:flutter/material.dart';
+
+import '../Widgets/GridText.dart';
+import '../Widgets/ImageHelper.dart';
+
+class UserPage extends StatelessWidget {
+  UserPage();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.lightBlue,
```

Comment:
Should we set the background and nav color to theme variables?

Reply:
Yes, this will be handled in a later PR.